templates/adp-backend-template-dotnet/skeleton/src/${{ values.dotnet_solution_name }}.Api/Program.cs

```diff
@@ -0,0 +1,116 @@
+using System.Diagnostics.CodeAnalysis;
+using ${{ values.dotnet_solution_name }}.Api.Config;
+using ${{ values.dotnet_solution_name }}.Api.Extensions;
+using ${{ values.dotnet_solution_name }}.Api.HealthChecks;
+using Asp.Versioning;
+using Azure.Identity;
+using Azure.Monitor.OpenTelemetry.AspNetCore;
+using Microsoft.AspNetCore.Diagnostics.HealthChecks;
+using Microsoft.OpenApi.Models;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+
+namespace ${{ values.dotnet_solution_name }}.Api
+{
+    [ExcludeFromCodeCoverage]
+    public static class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            ConfigureServices(builder);
+
+            var app = builder.Build();
+
+            // Configure the HTTP request pipeline.
+            if (app.Environment.IsDevelopment())
+            {
+                app.UseSwagger();
+                app.UseSwaggerUI();
+                app.UseDeveloperExceptionPage();
+            }
+            app.UseExceptionHandler();
+
+            app.UseHttpsRedirection();
+            app.UseAuthorization();
+            app.MapControllers();
+
+            app.UseHealthChecks("/healthy", new HealthCheckOptions()
+            {
+                Predicate = check => check.Name == "ready"
+            });
+            app.UseHealthChecks("/healthz", new HealthCheckOptions()
+            {
+                Predicate = check => check.Name == "live"
+            });
+            app.UseHealthChecks("/", new HealthCheckOptions()
```

Comment:
Do we definitely want health checks on the root path? This exact health check is available on /healthz so it seems a bit redundant.